pkgs/development/libraries/libproxy/default.nix

```diff
@@ -12,22 +12,34 @@ stdenv.mkDerivation rec {
     sha256 = "10swd3x576pinx33iwsbd4h15fbh2snmfxzcmab4c56nb08qlbrs";
   };
 
+  patches = [ ./files/apple-webkit-disable.patch ];
```

Comment:
You can just put it next to `default.nix` there probably will not be many patches. Ideally we would open a pull request against libproxy and just `fetchpatch`ed it.